### PR TITLE
Improve demo scripts

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

File: trillian/integration/ct_functions.sh [hunk 1]
```diff
@@ -21,6 +22,7 @@ CT_COMBINED_CFG=
 # If etcd and Prometheus are configured, it also populates:
 #  - ETCDISCOVER_PID : pid of etcd service watcher
 #  - PROMETHEUS_PID  : pid of local Prometheus server
+#  - PROMETHEUS_CFGDIR  : Prometheus configuration directory
```
Comment (Martin2112) [inline trillian/integration/ct_functions.sh]: nit: were these meant to line up horizontally?

File: trillian/integration/demo-script.sh [hunk 2]
```diff
@@ -95,3 +95,25 @@ echo 'Remember to kill off all of the jobs, so their (hard-coded) ports get free
 ./trillian/integration/ct_killall.sh
 echo '...but ct_killall does not kill the hammer'
 killall -9 ct_hammer
+
+
+# Other ideas to extend a linear demo:
```
Comment (Martin2112) [inline trillian/integration/demo-script.sh]: Seems like a different PR?

File: trillian/integration/ct_functions.sh (hunk 1 above)
Comment (daviddrysdale) [reply trillian/integration/ct_functions.sh]: Fixed.

File: trillian/integration/demo-script.sh (hunk 2 above)
Comment (daviddrysdale) [reply trillian/integration/demo-script.sh]: Same PR, different commit; I've adjusted the PR description.

**Pull request merged**